plugins/TEMPLATE.nix

```diff
@@ -1,11 +1,13 @@
 {
   lib,
-  helpers,
   config,
   pkgs,
   ...
 }:
-helpers.neovim-plugin.mkNeovimPlugin config {
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin config {
```

Comment:
Do we want to also demonstrate `types = lib.nixvim.nixvimTypes`?

#1978 can be responsible for updating this line to `inherit (lib) types`

Reply:
We should wait for this indeed.